deepmd/entrypoints/compress.py

```diff
@@ -34,9 +34,9 @@ def compress(
     """Compress model.
 
     The table is composed of fifth-order polynomial coefficients and is assembled from
-    two sub-tables. The first table takes the stride(parameter) as it's uniform stride,
-    while the second table takes 10 * stride as it's uniform stride. The range of the
-    first table is automatically detected by deepmd-kit, while the second table ranges
+    two sub-tables. The first table takes the step parameter as the domain's uniform step size,
+    while the second table takes 10 * step as it's uniform step size. The range of the
+    first table is automatically detected by the code, while the second table ranges
     from the first table's upper boundary(upper) to the extrapolate(parameter) * upper.
```

Comment:
I don't understand what is "upper", it is not in the parameter list.

Reply:
changed in the description.